### PR TITLE
Fixed topk op

### PR DESCRIPTION
**Pull request by wjj19950828**
# Create A Good Pull Request

> 下面的文字请保留在PR说明的最后面，并在提完PR后，根据实际情况勾选确认以下情况  

Please check the follow step before merging this pull request

- [x] Python code style verification
- [x] Review all the code diff by yourself
- [ ] All models(TensorFLow/Caffe/ONNX/PyTorch) testing passed
- [x] Details about your pull request, releated issues

If this PR add new model support, please update `model_zoo.md` and add model to out test model zoos(@wjj19950828)
- [ ] New Model Supported
- [x] No New Model Supported

## Do follow contributes
1、优化topk实现，增加k为attr的实现
2、solved https://github.com/PaddlePaddle/X2Paddle/issues/803

### CODE REVIEW

File: x2paddle/op_mapper/onnx2paddle/opset9/opset.py [hunk 1]
```diff
@@ -2577,27 +2577,44 @@ def assign_params(op_name, weights, weight_idx=0, suffix=''):
...
-                "{}_p{}".format(node.layer_name, 1)
-            ],
-            **layer_attrs)
+        is_k_attr = False
+        k = _const_weight_or_none(val_k)
+        if isinstance(k, (list, tuple, np.ndarray)):
+            k = k[0]
+        if k is not None:
+            is_k_attr = True
+            layer_attrs["k"] = k
```
Comment (jiangjiajun) [inline x2paddle/op_mapper/onnx2paddle/opset9/opset.py:2591]: 这里判断了k is not None，如果k是None会怎样？

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: x2paddle/op_mapper/onnx2paddle/opset9/opset.py (hunk 1 above)
Comment (wjj19950828) [reply x2paddle/op_mapper/onnx2paddle/opset9/opset.py:2591]: 这里主要分为两种情况：k不为None，作为属性传入；k为None，作为输入传入

修改了下写法，判断k is not None以及else，Done.

Comment (jiangjiajun) [reply x2paddle/op_mapper/onnx2paddle/opset9/opset.py:2591]: 这种代码，加上注释吧。 之前的注释确实太少，现在有这种逻辑的，都加上注释说明，避免以后再回过头看代码，不清楚这个逻辑是在干啥

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: x2paddle/op_mapper/onnx2paddle/opset9/opset.py (hunk 1 above)
Comment (wjj19950828) [reply x2paddle/op_mapper/onnx2paddle/opset9/opset.py:2591]: Done.

**Review by jiangjiajun — APPROVED** [review]

**Pull request merged**